delivery/media/watermarking/lib/media-delivery-watermarking.js

```diff
@@ -0,0 +1,2134 @@
+        if (length < 6) {
+            if (4 === length) return fromCharCode(a, b, c, d);
+            {
+                let e = src[position$1++];
+                return (128 & e) > 0 ? void (position$1 -= 5) : fromCharCode(a, b, c, d, e);
+            }
+        }
+        if (length < 8) {
+            let e = src[position$1++], f = src[position$1++];
+            if ((128 & e) > 0 || (128 & f) > 0) return void (position$1 -= 6);
+            if (length < 7) return fromCharCode(a, b, c, d, e, f);
+            let g = src[position$1++];
+            return (128 & g) > 0 ? void (position$1 -= 7) : fromCharCode(a, b, c, d, e, f, g);
+        }
+        {
+            let e = src[position$1++], f = src[position$1++], g = src[position$1++], h = src[position$1++];
+            if ((128 & e) > 0 || (128 & f) > 0 || (128 & g) > 0 || (128 & h) > 0) return void (position$1 -= 8);
+            if (length < 10) {
+                if (8 === length) return fromCharCode(a, b, c, d, e, f, g, h);
+                {
+                    let i = src[position$1++];
+                    return (128 & i) > 0 ? void (position$1 -= 9) : fromCharCode(a, b, c, d, e, f, g, h, i);
+                }
+            }
+            if (length < 12) {
+                let i = src[position$1++], j = src[position$1++];
+                if ((128 & i) > 0 || (128 & j) > 0) return void (position$1 -= 10);
+                if (length < 11) return fromCharCode(a, b, c, d, e, f, g, h, i, j);
+                let k = src[position$1++];
+                return (128 & k) > 0 ? void (position$1 -= 11) : fromCharCode(a, b, c, d, e, f, g, h, i, j, k);
+            }
+            {
+                let i = src[position$1++], j = src[position$1++], k = src[position$1++], l = src[position$1++];
+                if ((128 & i) > 0 || (128 & j) > 0 || (128 & k) > 0 || (128 & l) > 0) return void (position$1 -= 12);
+                if (length < 14) {
+                    if (12 === length) return fromCharCode(a, b, c, d, e, f, g, h, i, j, k, l);
+                    {
+                        let m = src[position$1++];
+                        return (128 & m) > 0 ? void (position$1 -= 13) : fromCharCode(a, b, c, d, e, f, g, h, i, j, k, l, m);
+                    }
+                }
+                {
+                    let m = src[position$1++], n = src[position$1++];
+                    if ((128 & m) > 0 || (128 & n) > 0) return void (position$1 -= 14);
+                    if (length < 15) return fromCharCode(a, b, c, d, e, f, g, h, i, j, k, l, m, n);
+                    let o = src[position$1++];
+                    return (128 & o) > 0 ? void (position$1 -= 15) : fromCharCode(a, b, c, d, e, f, g, h, i, j, k, l, m, n, o);
+                }
+            }
+        }
+    }
+}
+
+let f32Array = new Float32Array(1), u8Array = new Uint8Array(f32Array.buffer, 0, 4);
+
+new Array(4096);
```

Comment:
I think there's a comment about this in the CWT impl. Can this be removed?

Reply:
@evan-hughes This is coming directly from cbor-x library (https://github.com/kriszyp/cbor-x/blob/master/decode.js#L747) during rollup build. My suggestion would be to keep as it is as we do not know how it is being used internally by cbor-x. 
Also changing manually the output artifact might lead to inconsistency whenever library gets upgraded.

Reply:
Hence also in CWT module i have not removed it.